README.md

```diff
@@ -1,2 +1,63 @@
-# fxa-geodb
```

Comment:
Can you add a LICENSE section to this doc that just points to the license file? See https://github.com/mozilla/fxa-content-server/#license